File: Beginner/1008 (Salary)/1008.java
import java.io.IOException;
import java.util.Scanner;
import java.text.DecimalFormat;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);
        DecimalFormat df = new DecimalFormat("0.00");
        int number = ler.nextInt();
        int hours = ler.nextInt();
        double salary = ler.nextDouble();
        
        
        double result = (salary * hours);
        System.out.println("NUMBER = " + number);
        System.out.println("SALARY = U$ " + df.format(result));
        
    }
 
}

File: Beginner/1008 (Salary)/1008.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            int number = int.Parse(Console.ReadLine());
            int hours = int.Parse(Console.ReadLine());
            double salary = double.Parse(Console.ReadLine());

            Console.WriteLine("NUMBER = " + number );
            Console.WriteLine("SALARY = U$ " +string.Format("{0:0.00}", salary * hours));


        }
    }
}